node_modules/oae-authentication/lib/test/util.js

```diff
@@ -49,3 +50,72 @@ var assertUpdateAuthConfigSucceeds = module.exports.assertUpdateAuthConfigSuccee
         });
     });
 };
+
+/**
+ * Sign in with the google strategy. Note that this function will NOT assert whether
+ * authentication was successful
+ *
+ * @param  {String}         tenantHost              The host of the tenant on which to log in through google
+ * @param  {String}         [email]                 The email address of the user that will sign in. If null, no email will be returned in the mocked response
+ * @param  {Function}       callback                Standard callback function
+ * @param  {RestContext}    callback.restContext    The rest context that was used to sign in through google
+ * @param  {Response}       callback.response       The response that came back from the google callback endpoint
+ */
+var googleLogin = module.exports.googleLogin = function(tenantHost, email, callback) {
```

Comment:
Any reason the public exposure of this couldn't be `assertGoogleLoginSucceeds` and `assertGoogleLoginFails` ?